catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -68,29 +73,50 @@ const SearchForm = ({
     }
   }, [query]);
 
-  function updateUrl(unfilteredSearchResults) {
+  function updateUrl(unfilteredSearchResults, form) {
     const workType = searchParams.workType || [];
     const link = unfilteredSearchResults
       ? worksUrl({
           ...searchParams,
           // Override the defaultWorkType with [] if we're toggled to do so
+          // null => default filters
+          // [] => no filter
+          // [anything] => filter
           workType: workType.length === defaultWorkTypes.length ? [] : workType,
           query: inputQuery,
           page: 1,
+          // $FlowFixMe
```

Comment:
Is this because we need to check for `form.current`?

Reply:
`currentTarget` generally ensures  type safety which we should try for